src/Transports.AspNetCore/WebSockets/GraphQLWs/SubscriptionServer.cs

```diff
@@ -113,6 +122,69 @@ public override async Task OnMessageReceivedAsync(OperationMessage message)
         }
     }
 
+    /// <inheritdoc/>
+    [Obsolete($"Please use the {nameof(OnConnectionInitAsync)} and {nameof(OnKeepAliveLoopAsync)} methods instead. This method will be removed in a future version of this library.")]
+    protected override Task OnConnectionInitAsync(OperationMessage message, bool smartKeepAlive)
```

Comment:
The code has been carefully crafted to be fully backwards compatible, just in case of the rare instance that someone derived from this class to customize behavior.